docs/installation/post-install.rst

```diff
@@ -40,63 +40,81 @@ path for the partitions to use:
    apiVersion: storage.metalk8s.scality.com/v1alpha1
    kind: Volume
    metadata:
-     name: <node_name>-alertmanager
+     name: bootstrap-alertmanager
    spec:
      nodeName: <node_name>
      storageClassName: metalk8s-prometheus
-     rawBlockDevice:  # Choose a device with at least 1GiB capacity
-       devicePath: <device_path2>
+     sparseLoopDevice:
+       size: 1Gi
      template:
        metadata:
          labels:
            app.kubernetes.io/name: 'prometheus-operator-alertmanager'
    ---
 
-Once this file is created with the right values filled in, run the following
-command to create the *Volume* objects (replacing ``<file_path>`` with the path
+Once you have created this file with correct values entered, run the following
+command to create the volume objects (replacing ``<file_path>`` with the path
 of the aforementioned YAML file):
 
 .. code-block:: shell
 
-   root@bootstrap $ kubectl --kubeconfig /etc/kubernetes/admin.conf \
-                      apply -f <file_path>
+   root@bootstrap # kubectl --kubeconfig /etc/kubernetes/admin.conf apply -f <file_path>
```

Comment:
This was also 'wrong' in the original: we should at all times assume `KUBECONFIG` is exported and not pass `--kubeconfig ...` every time to reduce duplication / useless overhead.